site/src/pages/components/selectlist.mdx

```diff
@@ -139,7 +140,28 @@ The default behavior of the `<selectlist>` control mimics the behavior of the `<
 </selectlist>
 ```
 
-When doing so, the default button, `<selectedoption>`, and `<listbox>` are created for you.
+When doing so, the default button, `<selectedvalue>`, and `<listbox>` are created for you. As with the current `<select>` element, the initial `<option>` will be returned as the `<selectedvalue>`.
```

Comment:
Isn't it `<selectoption>` rather than `<selectedvalue>`?

Reply:
oh that's going to trip me up forever.  just like saying "hide" and "show" instead of "open" and "close". thank you